src/App.tsx

```diff
@@ -53,18 +56,21 @@ const App: React.FC = () => (
       </>
     )}
 
-    <UserProvider>
-      <CookiesProvider>
-        <ThemeProvider theme={mdTheme}>
-          <CssBaseline />
-          <Router>
-            <Routes>
-              <Route path="/*" element={<ControlPanel />} />
-            </Routes>
-          </Router>
-        </ThemeProvider>
-      </CookiesProvider>
-    </UserProvider>
+    <QueryClientProvider client={queryClient}>
```

Comment:
added useQuery's provider allowing the whole app to use the same queryClient instance